### ISSUE TITLE
Crash when using obsidian pipes

### ISSUE THREAD

**Issue by tahg** [BuildCraft/BuildCraft#3651]
As this is in issue with a library function, it's probably not limited to just this specific cause.  Code references a constructor that only exists client side.  (BC 7.99.3)

```
Captured exception:
java.lang.NoSuchMethodError: net.minecraft.util.math.AxisAlignedBB.<init>(Lnet/minecraft/util/math/Vec3d;Lnet/minecraft/util/math/Vec3d;)V
at buildcraft.lib.misc.BoundingBoxUtil.makeAround(BoundingBoxUtil.java:59)
at buildcraft.transport.pipe.behaviour.PipeBehaviourObsidian.getSuckingBox(PipeBehaviourObsidian.java:131)
at buildcraft.transport.pipe.behaviour.PipeBehaviourObsidian.suckEntity(PipeBehaviourObsidian.java:117)
at buildcraft.transport.pipe.behaviour.PipeBehaviourObsidian.onTick(PipeBehaviourObsidian.java:82)
at buildcraft.transport.pipe.Pipe.onTick(Pipe.java:209)
at buildcraft.transport.tile.TilePipeHolder.func_73660_a(TilePipeHolder.java:200)
at net.minecraft.world.World.func_72939_s(World.java:1802)
at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:621)
at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:720)
at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:384)
at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:624)
at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:482)
at java.lang.Thread.run(Thread.java:745)
```

**Comment by AEnterprise** [comment]
the latest version of BC is 7.99.4 so you might want to update to that, though i don't think this particular bug is fixxed in there

**Comment by AlexIIL** [comment]
Fixed in [7.99.5](https://www.mod-buildcraft.com/buildcraft-7995-alpha.html)
